src/FeedBackServo.cpp

```diff
@@ -108,11 +138,78 @@ void FeedBackServo::rotate(int degree, int threshold)
     }
 }
 
+[[deprecated("Use getAngle() for future support.")]]
 int FeedBackServo::Angle()
 {
     return getAngle();
 }
 
+
+// Sets the current direction of rotation.
+// Constants::Direction::FORWARD will rotate in the counterclockwise direction.
+// Constants::Direction::BACKWARD wil rotate in the clockwise direction.
+void FeedBackServo::setDirection(Constants::Direction direction)
+{
+    switch(direction)
+    {
+        case Constants::Direction::FORWARD:
+            direction_ = false; // Counter-clockwise
+            break;
+        case Constants::Direction::BACKWARD:
+            direction_ = true; // Clockwise
+            break;
+    }
+}
+
+// Determines whether current mode facilitates desired form of motion.
+bool FeedBackServo::checkMode(Constants::Mode mode){    
+    return mode_ != mode;
+
+}
+
+// Rotate servo according to a given percent representing power output.
+// Direction can be toggled through setDirection().
+// Note: Recommended for speed control. Does NOT utilize position control.
+void FeedBackServo::move(float percent){
+
+    if(checkMode(Constants::Mode::VELOCITY_CONTROL)) return;
+
+    float speed;
+    if (percent == 0.0f){
+        speed = 0.0f;
+    } else {
+        speed = direction_ ? Constants::LOWER_NEUTRAL_BOUND - Constants::POWER_CONVERSION * percent : 
+                             Constants::UPPER_NEUTRAL_BOUND + Constants::POWER_CONVERSION * percent;
+    }
+    parallax_.writeMicroseconds(speed);
+}
+
+// Rotate servo according to a desired rotations per minute.
+// Direction can be toggled through setDirection().
+// Note: Recommended for speed control. Does NOT utilize position control.
+void FeedBackServo::move(unsigned int rpm){
+
+    if(checkMode(Constants::Mode::VELOCITY_CONTROL)) return;
+
+    float speed;
+    switch (rpm){
+        case 0:
+            speed = 0;
+            break;
+        default:
+            speed = direction_ ? Constants::LOWER_NEUTRAL_BOUND - Constants::RPM_CONVERSION * rpm : 
+                                 Constants::UPPER_NEUTRAL_BOUND + Constants::RPM_CONVERSION * rpm;
+            break;
+    }
+    parallax_.writeMicroseconds(speed);
+}

```

Comment:
Please consider to add value constraints for tControl within the range of 1280 to 1720 before calling `parallax_.writeMicroseconds(speed)` to prevent improper behavior.